src/tasks/daily.ts

```diff
@@ -365,6 +368,22 @@ export function configureSnojo(): void {
   }
 }
 
+function determineFreeEmbezzlerBanish(): void {
+  if (freeBanishesChecked) return;
+  const useFreeBanishes =
+    getBanishedMonsters().get($item`ice house`) !== $monster`fluffy bunny` &&
+    gregLikeFightCount() < 120 && // 60 turns of banish from mafia middle finger ring, and 30 x 2 from two snokebombs
```

Comment:
So one issue here is that we don't _just_ care about the greglike embezzlers, we care about all embezzlers we do before our final gregory. That includes profchain, for example, as well as our chain starter and probably a few interpolated digitizes